File: scbp-service-gateway/src/main/java/com/halower/scbp/gateway/ScbpApiGatewayApplication.java
package com.halower.scbp.gateway;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
import org.springframework.cloud.netflix.zuul.EnableZuulProxy;
import org.springframework.context.annotation.ComponentScan;

@EnableZuulProxy
@EnableDiscoveryClient
@SpringBootApplication
public class ScbpApiGatewayApplication {

	public static void main(String[] args) {
		SpringApplication.run(ScbpApiGatewayApplication.class, args);
	}
}

File: scbp-gateway-server/src/main/kotlin/com/halower/scbp/apigateway/ScbpApiGatewayApplication.kt
package com.halower.scbp.apigateway

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient
import org.springframework.cloud.netflix.zuul.EnableZuulProxy


@EnableZuulProxy
@EnableDiscoveryClient
@SpringBootApplication
class ScbpApiGatewayApplication

fun main(args: Array<String>) {
    runApplication<ScbpApiGatewayApplication>(*args)
}
